dbms/programs/client/Client.cpp

```diff
@@ -462,6 +462,11 @@ class Client : public Poco::Util::Application
             if (rl_initialize())
                 throw Exception("Cannot initialize readline", ErrorCodes::CANNOT_READLINE);
 
+#if RL_VERSION_MAJOR >= 7
+            // Use bracketed paste whenever possible
```

Comment:
Better to explain what is bracketed paste instead of duplication of var name in comment.

Reply:
Ok.